pkg/api/jobrunintervals/job_run_intervals.go

```diff
@@ -15,26 +15,46 @@ import (
 )
 
 // JobRunIntervals fetches intervals for a given job run.
-func JobRunIntervals(gcsClient *storage.Client, dbc *db.DB, jobRunID int64, logger *log.Entry) (*apitype.EventIntervalList, error) {
+// It fetches from the prow job's GCS bucket path contructed using the prow job name and jobID
```

Comment:
Godoc standard requires the function name to be the first word in the comment, i.e. this should remain "JobRunIntervals fetches", not "It fetches".